chain-core/src/property.rs

```diff
@@ -367,6 +367,13 @@ impl<T: Serialize> Serialize for &T {
     }
 }
 
+pub struct NodeId {
+    pub high: u64,
+    pub low: u64,
+}
+
+pub trait Gossip {}
```

Comment:
This is also a network thing, if it's needed at all.

Reply:
and add `Serialize + Deserialize` too as dependency of this trait